lib/ansible/plugins/doc_fragments/panos.py

```diff
@@ -25,3 +25,221 @@ class ModuleDocFragment(object):
         type: str
         default: admin
 '''
+
+    PROVIDER = r'''
+options:
+    provider:
+        description:
+            - A dict object containing connection details.
+        version_added: 2.8
```

Comment:
```suggestion
        version_added: '2.8'
```

Reply:
Resolved in 0595acc4f78f55103e406b523b7fd67115105162